tools/depends/target/libmicrohttpd/Makefile

```diff
@@ -35,13 +35,13 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	cd $(PLATFORM); $(CONFIGURE)
 
 $(LIBDYLIB): $(PLATFORM)
-	$(MAKE) -C $(PLATFORM)/src/microhttpd
-	$(MAKE) -C $(PLATFORM)/src/include install
+	$(MAKE) -j1 -C $(PLATFORM)/src/microhttpd
+	$(MAKE) -j1 -C $(PLATFORM)/src/include install
```

Comment:
yeah might not be an issue on jenkins but had it with -j12 on an i7 - i'll try to figure out why